build/AzurePipelinesTemplates/ProjectReunion-PublishProjectOutput-Steps.yml

```diff
@@ -9,6 +9,18 @@ steps:
       filePath: build\CopyFilesToStagingDir.ps1
       arguments: -BuildOutputDir '$(buildOutputDir)' -PublishDir '$(Build.ArtifactStagingDirectory)\${{ parameters.artifactName }}' -NugetDir '$(Build.ArtifactStagingDirectory)\FullNuget' -Platform '$(buildPlatform)' -Configuration '$(buildConfiguration)'
 
+  - script: |
```

Comment:
are these debugging output that you had intended to remove?

Reply:
Yep. Gone in latest